vault/src/issue.rs

```diff
@@ -151,6 +151,7 @@ pub async fn add_keys_from_past_issue_request(
     let mut scanning_status = RescanStatus::get(vault_id, db)?;
     tracing::info!("initial status: = {scanning_status:?}");
 
+    // TODO: remove filter since we use a shared wallet
```

Comment:
Can you address this in this pr? Currently we're calling `add_keys_from_past_issue_request` from each vault. Each one will try to rescan a different range, and I'm afraid that something may go wrong when this happens. So change request is: make sure we call `add_keys_from_past_issue_request` a single time, and that it processes the issues for all vaultids. We'll need to clear the cached rescan status as well, we can do this by renaming the key, like I did before